src/index.html

```diff
@@ -8,7 +8,52 @@
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="home">
+        <img class="img" src="images/logo.png"
+        alt="Moyo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="is_active nav__link" href="#apple">Apple</a></li>
```

Comment:
Fix formatting:

```
<li class="nav__item">
  <a class="is_active nav__link" href="#apple">Apple</a>
</li>
```

Reply:
Check other similar cases

Reply:
I can’t understand what’s wrong with this formatting ?
What should I do?